devtools/README

```diff
@@ -174,6 +174,21 @@ ______________________________
     Achievements lists.
 
 
+make_class.py
+-------------------
+    Tool that adds all the boilerplate for a new C++ class inside an engine
+	Examples:
+
+	$ make_class.py scumm . LeChuck
+	Make new class `LeChuck` in the scumm engine root (engines/scumm/).  Creates
+	boilerplate le_chuck.cpp and le_chuck.h containing class Scumm::LeChuck and
```

Comment:
This example is a bit confusing: how the script comes with `le_chuck` (with underscore) from `LeChuck`?

Reply:
It converts camelcase class names to lowercase with underscores to follow the convention, I could mention that explicitly.

Reply:
Done in 8d0aa01b163.